recipes/connectors.rb

```diff
@@ -18,26 +18,26 @@
 
 yum_repository 'mysql-connectors-community' do
   description node['yum']['mysql-connectors-community']['description'] unless node['yum']['mysql-connectors-community']['description'].nil?
-  baseurl node['yum']['mysql-connectors-community']['baseurl'] if  node['yum']['mysql-connectors-community']['baseurl'].nil?
+  baseurl node['yum']['mysql-connectors-community']['baseurl'] unless  node['yum']['mysql-connectors-community']['baseurl'].nil?
```

Comment:
If you clear out the extra spaces between unless and node here and on lines 39 and 40, the find/replace that caused this issue won't miss these lines next time.